src/cli/cli.cpp

```diff
@@ -1520,6 +1520,11 @@ void Interpreter::ProcessScan(int argc, char *argv[])
     AppendResult(error);
 }
 
+void Interpreter::s_HandleActiveScanResult(otActiveScanResult *aResult)
+{
+    Uart::sUartServer->mInterpreter.HandleActiveScanResult(aResult); // BUGBUG - Callbacks must provide a context
```

Comment:
All registered callbacks must have a callback context pointer. This BUGBUG was a note to myself to make another change to add this support.